### PR TITLE
*: bump to 2.0.2

### PR DESCRIPTION
**Pull request by BusyJay**
* Support configuring more gRPC related parameters
* Support configuring the timeout range of leader election
* Fix the issue that the Raft log is not printed
* Fix the issue that obsolete learner is not deleted
* Fix the issue that the snapshot intermediate file is mistakenly deleted

### CODE REVIEW

**Comment by BusyJay** [comment]
/run-all-tests tidb=release-2.0 pd=release-2.0 tidb-test=release-2.0

**Comment by zhangjinpeng1987** [comment]
LGTM

**Comment by breeswish** [comment]
LGTM

File: Cargo.toml [hunk 1]
```diff
@@ -105,6 +105,8 @@ optional = true
 
 [replace]
 "protobuf:1.4.1" = { git = "https://github.com/stepancheg/rust-protobuf.git" }
+# TODO: use release version instead once tikv is protobuf 1.6 compatible.
+"raft:0.2.0" = { git = "https://github.com/pingcap/raft-rs.git", branch = "timeout-range" }
```
Comment (hicqu) [inline Cargo.toml]: This can be removed.

**Comment by hicqu** [comment]
LGTM.

**Pull request merged**